frame/elections-phragmen/src/lib.rs

```diff
@@ -460,7 +466,6 @@ decl_module! {
 			let target = T::Lookup::lookup(defunct.who)?;
 
 			ensure!(reporter != target, Error::<T>::ReportSelf);
-			ensure!(Self::is_voter(&reporter), Error::<T>::MustBeVoter);
```

Comment:
Note that before because we had to do (potential) slashing the origin (reporter) must have been a voter. Now it can be anyone.